packages/neo-one-client/src/provider/neoone/NEOONEDataProvider.js

```diff
@@ -283,6 +285,10 @@ export default class NEOONEDataProvider implements DataProvider {
     return this.testInvoke(testTransaction.serializeWire().toString('hex'));
   }
 
+  startConsensusNow(): void {
```

Comment:
This should return a `Promise<void>`. You're not properly waiting on `startConsensusNow` - if it throws an error it will never propagate to the caller.